projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/pojo/Analysis.java

```diff
@@ -0,0 +1,78 @@
+package org.batfish.datamodel.pojo;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import java.util.Map;
+import java.util.Objects;
+import javax.annotation.Nullable;
+import org.batfish.datamodel.questions.Question;
+
+/**
+ * The {@link Analysis Analysis} is an Object representation of the analysis for Batfish service.
+ *
+ * <p>Each {@link Analysis Analysis} contains a name and a mapping from question name to {@link
+ * Question question} for questions in the Analysis.
+ */
+public class Analysis {
+  private static final String PROP_NAME = "name";
+  private static final String PROP_QUESTIONS = "questions";
+
+  private String _name;
+  private @Nullable Map<String, Question> _questions;
+
+  public Analysis(@JsonProperty(PROP_NAME) String name) {
+    this(name, null);
```

Comment:
Should I use empty hashmap here? I feel when we creating an empty Analysis with its name(same as creating empty folder today), it makes more sense to have an empty map of questions.